src/main/kotlin/org/arend/refactoring/changeSignature/ArendChangeSignatureDialog.kt

```diff
@@ -55,9 +49,7 @@ class ArendChangeSignatureDialog(
     override fun createVisibilityControl() = object : ComboBoxVisibilityPanel<String>("", arrayOf()) {}
 
     private fun createTypeCodeFragment(function: PsiElement): PsiCodeFragment {
-        // or `return PsiTypeCodeFragmentImpl(function.project, true, "dummyName", function.text, 0, myDefaultValueContext)`
-        return PsiTypeCodeFragmentImpl(function.project, true, "dummyName", function.text, 0, function.context)
+        return PsiTypeCodeFragmentImpl(project, true, "fragment.ard", function.text, 0, myDefaultValueContext)
```

Comment:
Why not use `ArendPsiFactory` to create Psi fragments?

Reply:
I thought there would be problems casting to `PsiCodeFragment`, but in this case everything is ok, rewrote using `ArendPsiFactory`.

Reply:
cool